Sdtrig.tex

```diff
@@ -275,6 +275,27 @@ \subsection{Combined Accesses}
 loads of size SEW (selected element width), and {\tt cm.push} should be treated
 as if it performed multiple stores of size XLEN.
 
+\subsection{Cache Operations}
+
+As far as triggers are concerned, caches don't exist. Cache operations that
+change the apparent contents of memory should match triggers as if those
+operations did XLEN-bit stores to memory. No other cache operations should match
+triggers.
```

Comment:
If memory happened to already contain 0 and somebody does a cbo.zero then it doesn't change the apparent contents of memory.  I assume that we agree that address triggers on cbo.zero should always match without doing any data comparison.

Should we just spell out all the cases from the cache management spec instead of having this brief description?  We might elaborate on the Zicbom/Zicbop cases to say that what they generally say is "optional" means that actual changes to the apparent contents of memory should match like stores and non-changes (like flushes or prefetches) are not required to or are encouraged not to match.  (I don't think that we need to require that prefetches must not match.  Maybe it's much easier for some implementation to match on those or somebody has some special reason to care about prefetches.)